README.md

```diff
@@ -57,8 +73,9 @@ Currently, the process for releasing a new version of our package is as follows:
 2. Merge your pull request.
 3. Publish a [GitHub release](https://github.com/edx/studio-frontend/releases). Make sure to prefix the version number with `v`, as in `v2.3.4`.
 3. `git checkout master` and `git pull`. Ensure your current directory is cleaned, with no outstanding commits or files. As an extra precaution, you can `rm -rf node node_modules` and `npm install` prior to publishing.
-4. Be a member of the correct edX and npm orgs, and be logged in. All of @edx/educator-dahlia should be set up, and others shouldn't need to be publishing this package.
-5. Run `npm publish`.
+4. Make sure that build production files in /dist will be included in the release by running `npm run build`.
```

Comment:
I added this because I noticed that the npm prepublish step failed for me inside the studio-frontend container. I think it is because it was running as root and NPM doesn't like that.

How much trouble would it be to log into the studio-frontend container not as root (e.g. devstack uses "edxapp" user)?